docs/pages/lighting-setups.rst

```diff
@@ -0,0 +1,197 @@
+Working with Lights
+###################
+
+:summary: This tutorial demonstrates the creation and manipulation of LightSetups in Habitat-sim.
```

Comment:
We should make this sentence broadly accessible. 

This tutorial demonstrates lighting of scenes and objects in Habitat -- creation, placement, configuration, manipulation of light sources and shaders.